### PR TITLE
Represent payment info in milestone conversation

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to Giveth/giveth-dapp#1918, Giveth/giveth-dapp#1847

it's needed more tests but because of this issue #424, I couldn't test fully integrated, but I think it works.

The conversation are like this 

Screen Shot 1399-12-29 at 10 36 45

And the transactionLink of conversation is in etherscan net

### CODE REVIEW

**Comment by aminlatifi** [comment]
@mohammadranjbarz I liked to resolve the conflict, but I left it for you when you come back

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz I liked to resolve the conflict, but I left it out for you when you come back since I guess it would cause some issue

@aminlatifi  done

File: src/utils/bridgeMonitor.js [hunk 1]
```diff
@@ -110,22 +157,39 @@ const inquiryAndUpdateDonationStatusFromBridge = async ({ app, donation }) => {
...
-          DonationBridgeStatus.PAID,
-          DonationBridgeStatus.CANCELLED,
-          DonationBridgeStatus.EXPIRED,
-        ],
-      },
+      $or: [
+        {
+          ...query,
+          // 3 * 24 * 60 * 60 * 1000 means 3 days
+          createdAt: { $lte: new Date(Date.now() - 3 * 24 * 60 * 60 * 1000) },
```
Comment (aminlatifi) [inline src/utils/bridgeMonitor.js:180]: Do we still need this?

Comment (mohammadranjbarz) [reply src/utils/bridgeMonitor.js:180]: @aminlatifi
I think we can get rid of this condition, 
if you agree I remove it

Comment (aminlatifi) [reply src/utils/bridgeMonitor.js:180]: It's not a big deal, let's keep it

**Pull request merged**